lib/ui/app_bar.dart

```diff
@@ -11,21 +12,22 @@ import 'package:farmsmart_flutter/data/firebase_const.dart';
 
 // We define here generic margins for the app
 abstract class CustomAppBar {
-  static AppBar build(int currentHomeTab, Function goToPrivacyPolicy) {
+  static AppBar build(BuildContext context, int currentHomeTab, Function goToPrivacyPolicy) {
+    FarmsmartLocalizations localizations = FarmsmartLocalizations.of(context);
```

Comment:
This will change as a view model will be added, but for now we can use this as a working example for the localisations. 👍

Reply:
Indeed I chose the appbar for demo since this piece has not been worked on anyway.